torch/csrc/jit/export.cpp

```diff
@@ -645,8 +626,22 @@ void ModuleEncoder::EncodeTensor(
         /* storageOffset = */ 0,
         /* size = */ { tensor.numel() },
         /* strides = */ { 1 })
-    .toBackend(at::kCPU);
-    raw_data_export_map_[name] = t;
+      .toBackend(at::kCPU);
```

Comment:
Instead of making a new tensor, we should just grab the data directly out of the storage object. Note: `tensor.numel()` is not necessarily the length of the storage! It might point somewhere in the middle of the storage with a non-zero storage offset.

Reply:
I think I need to copy CUDA tensors to CPU, and that requires me to create this tensor. Otherwise, the copied tensor may have a new storage.